### DISCUSSION TITLE
Incorrect aspect ratio for CPS1 games in MAME 2003 core

### DISCUSSION THREAD

**Discussion by svera** [OnionUI/Onion#1381] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2.3

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

CPS1 games (and probably CPS2 and CPS3) are shown with horizontal black bars if you use the default MAME 2003 core, as they are using a wrong aspect ratio. FBNeo core load these games with the right aspect ratio (4:3), filling the whole display of the Miyoo Mini Plus.

### Steps to Reproduce

Load any CPS1 game, for example `1944`, with the MAME 2003 core. The game will be shown with black bars at the top and bottom.

### Provide any additional context or information that may help us investigate your issue

Nothing.

**Comment by Aemiii91** [comment]
Our default core for CPS1 is `fbalpha2012_cps1` ?

**Reply by svera** [reply]
@Aemiii91 it is using MAME 2003 plus, loaded `1944` from the arcade folder.

**Reply by Aemiii91** [reply]
CPS1 does not go in arcade folder.

https://onionui.github.io/docs/emulators/cps-1
